CHANGELOG.rst

```diff
@@ -2,6 +2,60 @@
 CHANGELOG
 =========
 
+2.8.0
+=====
+
+**ENHANCEMENTS**
+
+* Print a warning when using NICE DCV on micro or nano instances.
+* Add support for S3 URI in custom_chef_cookbook.
+* Make `pcluster update` command safe.
+* Support DCV in China partition.
+* Use describe_instance_types to validate whether an instance type is EFA-enabled.
+* Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+* Support EFS in the China partition.
+* Enable configuration of the automatic backup features of FSx file systems.
+* Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
+  CentOS 6.
+* Install glibc-static, which is required to support certain options for the Intel MPI compiler.
+
+**CHANGES**
+
+* Migrate S3 URLs to Virtual Hosted Style Access.
+* Remove hardcoded account IDs from util/generate-ami-list.py.
+* Avoid validation for ParallelCluster required additional IAM policies.
```

Comment:
It doesn't sound good, is this a bug fix? what's the purpose of the change

Reply:
It's to avoid extra API calls for IAM policies we already know are valid. I'll reword it.

"Avoid validation of IAM policies known to be valid."